types/lib/users.d.ts

```diff
@@ -22,13 +23,90 @@ export interface CreateResponse extends BaseResponse {
     phone_id: string;
     status: string;
 }
-export interface GetResponse extends BaseResponse {
+interface User {
     user_id: UserID;
     name: Name;
     emails: Email[];
     phone_numbers: PhoneNumber[];
     webauthn_registrations: WebAuthnRegistration[];
+    providers: OAuthProvider[];
     status: string;
+    created_at: Date;
```

Comment:
Similar re-ordering to above if we would like to do that.

Reply:
done (btw, `types` and `dist` are both auto-generated from the build)